libraries/botbuilder-dialogs/src/dialogContext.ts

```diff
@@ -150,6 +150,20 @@ export class DialogContext {
         return this.context.turnState.get(DialogTurnStateConstants.dialogManager);
     }
 
+    /**
+     * Obtain the CultureInfo in DialogContext.
+     * @returns a locale string.
+     */
+    public getLocale(): string
```

Comment:
I don't think we want to hang locale off of dialogContext.

Reply:
this isn't creating a new property in dialogContext and duplicating the data from turnContext. This is a helper method to help the places where dc is available to get locale easier. 

previously, we have this helper function inside LGTemplateGenerator where it's used, then following @gabog's suggestion, we make it an extension method in dc, incase elsewhere we need it. 

do you think we should keep it inside LGTemplateGenerator and leave dc clear, which is also OK, i think.

Reply:
Sorry, I did not have the context for this change.  Thank you for the explanation @boydc2014